docs/source/user_manual/quickstart.rst

```diff
@@ -5,11 +5,11 @@
 Quickstart
 ##########
 
-+----------------------------------------+--------------------------------------+
-|.. image::  images/simple_line.png      |.. image::  images/scalar_function.png|
-|   :height: 300 px                      |   :height: 300 px                    |
-|   :align: center                       |   :align: center                     |
-+----------------------------------------+--------------------------------------+
++--------------------------------------+------------------------------------------+
+|.. image::  ../images/simple_line.png |.. image::  ../images/scalar_function.png |
```

Comment:
im not sure why the built docs point me to `simple_line2.png` instead of just `simple_line.png`

Reply:
`simple_line2.png`? I am not seeing that existing... 🤔 
There is a `simple_line.png` in both `docs/source/images` and `docs/source/usee_manual/images`.  I believe they are different sizes, and the one in `docs/source/images` matches the size of `scalar_function.png`.  Do the two images appear the same size for you?  They were for me